Public/Src/FrontEnd/Script/Evaluator/ContextTree.cs

```diff
@@ -44,6 +44,11 @@ public sealed class ContextTree : IDisposable
         /// </summary>
         public GlobalConstants Constants { get; private set; }
 
+        /// <summary>
+        /// CommonConstants

```

Comment:
nodoc?

Reply:
Or at least "Common constants" :)))

---
In reply to: [274539409](https://github.com/Microsoft/BuildXL/pull/127#discussion_r274539409) [](ancestors = 274539409)

Reply:
will do in next PR